tasks/build.rake

```diff
@@ -45,6 +45,7 @@ end
 # Only building source, but not gemspecs, version file, etc.
 task 'build:aws-sdk-sso' do
   sso = BuildTools::Services.service('sso')
+  sso.gem_dependencies.clear
```

Comment:
I'm a little wary about this - If someone ONLY installs/requires the aws-sdk-sso gem, that case still needs to work.